src/video/SDL_surface.c

```diff
@@ -68,13 +68,13 @@ SDL_CalculatePitch(Uint32 format, size_t width, SDL_bool minimal)
     return pitch;
 }
 
-/* TODO: In SDL 3, drop the unused flags and depth parameters */
+/* TODO: In SDL 3, drop the unused flags */
```

Comment:
Maybe it's out of the scope of this PR but this should be done eventually too, right? I suppose it wasn't captured in a github issue because it doesn't say SDL 2.1 here.